x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_edit_extension.tsx

```diff
@@ -32,47 +28,27 @@ import {
 } from '../../../../../../common/endpoint/types';
 import { useKibana } from '../../../../../common/lib/kibana';
 import { useNavigateToAppEventHandler } from '../../../../../common/hooks/endpoint/use_navigate_to_app_event_handler';
+import { IntegrationPolicyEditExtensionComponentProps } from '../../../../../../../fleet/common/types/ui_extensions';
```

Comment:
hopefully this can be imported from `fleet/public`?

Reply:
yes, it is now after I moved the types over to public. Thanks for the feedback.